crates/bevy_sprite/src/bundle.rs

```diff
@@ -10,11 +10,16 @@ use bevy_render::{
 };
 use bevy_transform::components::{GlobalTransform, Transform};
 
+/// A [`Bundle`] of components for drawing a single sprite from an image.
 #[derive(Bundle, Clone, Default)]
 pub struct SpriteBundle {
+    /// Specifies the rendering properties of the sprite, such as color tint and flip
     pub sprite: Sprite,
+    /// The local transform of the sprite, relative to its parent
     pub transform: Transform,
+    /// The absolute transform of the sprite. This should generally not be written to directly.
     pub global_transform: GlobalTransform,
+    /// A reference-counted handle to the image asset to be drawn
```

Comment:
```suggestion
    /// A reference-counted handle to the image asset to be drawn.
```

For consistency, I suggest to also add punctuation to the remaining fields.